playbooks/luna-devel.yml

```diff
@@ -2,9 +2,6 @@
 - import_playbook: katello_devel.yml
   vars:
     foreman_installer_options:
-      - "--katello-devel-github-username '{{ katello_devel_github_username }}'"
-      - "--katello-devel-enable-ostree=true"
-      - "--katello-devel-extra-plugins theforeman/foreman_remote_execution"
```

Comment:
Why do you drop REX here?

Reply:
Because it is already set in [katello_devel playbook](https://github.com/theforeman/forklift/blob/master/roles/katello_devel/meta/main.yml#L23) and installer is failing when you specify `--katello-devel-extra-plugins theforeman/foreman_remote_execution` twice.